### PR TITLE
Feat/remove http middlewares

### PR DESCRIPTION
**Pull request by Sc0ra**

### CODE REVIEW

**Review by GuillaumeLagrange — APPROVED** [review]
lgtm to me, :crab: :crab: :crab:  middy is cone :crab: :crab: :crab:

File: packages/serverless-helpers/src/helpers/index.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-export { applyHttpMiddlewares } from './applyHttpMiddlewares';
```
Comment (adriencaccia) [inline packages/serverless-helpers/src/helpers/index.ts:1]: This commit is breaking, so you need to add a `!` character in the commit message, like so:
`feat(serverless-helpers)!: remove applyHttpMiddlewares helper`

File: user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md [hunk 2]
```diff
@@ -316,11 +316,40 @@ The `getLambdaHandler` is only a pass-through adding typing capabilities, withou
 This method is not recommended, use `getHandler` when possible.
 :::
 
-In order to safely use `getLambdaHandler` combine it with `applyHttpMiddlewares`
+In order to safely use `getLambdaHandler`, you need to define a custom middleware than you can use to wrap your handler.
+Here is an implementation example:
+
+```ts
```
Comment (adriencaccia) [inline user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md:322]: You should add the `title="path/to/file.ts` to make it easier to understand where this file is located/what it is

File: user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md [hunk 3]
```diff
@@ -316,11 +316,40 @@ The `getLambdaHandler` is only a pass-through adding typing capabilities, withou
...
+  middyfiedHandler.use(jsonBodyParser());
+  middyfiedHandler.use(jsonValidator({ inputSchema, outputSchema }));
+
+  middyfiedHandler.use(httpErrorHandler());
+
+  return middyfiedHandler;
+};
+```
 
 ```ts
```
Comment (adriencaccia) [inline user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md:350]: Idem, can you add it here also?

**Review by adriencaccia — CHANGES_REQUESTED** [review]

**Pull request merged**